README.md

```diff
@@ -316,6 +316,14 @@ Type: `Function`
 
 A function that receives two arguments, the count of available options and the return value of `tStatusSelectedOption`, and should return the text used in the accessibility hint to indicate which options are available and which is selected.
 
+#### `tAssistiveHint` (default: `() => 'When autocomplete results are available use up and down arrows to review and enter to select.  Touch device users, explore by touch or with swipe gestures.'`)
```

Comment:
I've noticed in the other language we have stripped the end period, should we keep this new additional message consistent with this?

Reply:
As discussed :) .. it seems to be a specific to the aria live region announcement that the ending period is announced as 'dot'
The hint here is read ok, and the full stop & comma introduce the expected pauses in speech.